absl/logging/__init__.py

```diff
@@ -565,7 +565,11 @@ def find_log_dir_and_names(program_name=None, log_dir=None):
 
   actual_log_dir = find_log_dir(log_dir=log_dir)
 
-  username = getpass.getuser()
+  try:
+    username = getpass.getuser()
+  except:
```

Comment:
Instead of catching everything, please just catch the necessary exception type

Reply:
Done.
The python doc doesn't specify the exceptions it throws, so I caught the exception I got.
https://docs.python.org/3/library/getpass.html#getpass.getuser